pulsar-common/src/main/java/org/apache/pulsar/common/topics/TopicCompactionStrategy.java

```diff
@@ -60,17 +65,27 @@ public interface TopicCompactionStrategy<T> {
      */
     boolean shouldKeepLeft(T prev, T cur);
 
-    static TopicCompactionStrategy load(String topicCompactionStrategyClassName) {
+    default void handleSkippedMessage(String key, T cur) {
+    }
+
+
+    static TopicCompactionStrategy load(String tag, String topicCompactionStrategyClassName) {
```

Comment:
We should not cherry-pick this public API change to branch-3.0 @Technoboy-

Reply:
This API should only use for extensible load managers. I added an annotation to mark this as private, https://github.com/apache/pulsar/pull/20858 , please help take a look at this PR.